src/SmoFacade/Server.cs

```diff
@@ -122,7 +122,7 @@ public Database Database(string dbName)
     public void Restore(IEnumerable<BackupMetadata> backupOrder, string databaseName,
       Func<string, string> fileRelocation = null)
     {
-      var restore = new Restore { Database = databaseName, NoRecovery = true };
+      var restore = new Restore { Database = databaseName, NoRecovery = true, RetryFailedQueries = true };

```

Comment:
Why set retry failed queries here? Are you just being explicit and it was already set by default?

Reply:
boolean default is false. I thought this would be helpful, perhaps not?

Reply:
SMO default is true for this property. 
Helpful would depend on how the retry happens. If it'd mean a 4 hour timeout could try again and fail the job after 8 hours. May or may not be good.